preferencesfx/src/main/java/com/dlsc/preferencesfx/model/PreferencesFxModel.java

```diff
@@ -173,7 +178,7 @@ public Category loadSelectedCategory() {
   }
 
   /**
-   * Saves all of the values of the settings using a {@link StorageHandler}.
+   * Saves all of the values of the settings using a {@link StorageHandlerImpl}.
```

Comment:
should still be "StorageHandler", since if an own implementation of the `StorageHandler` is being used, it may not be called `StorageHandlerImpl`